### ISSUE TITLE
webui: rule content style

### ISSUE THREAD

**Issue by williballenthin** [mandiant/capa#2217]
Given this example rendering today:
![image](https://github.com/user-attachments/assets/0f258f69-f10b-4bf1-a8c5-904e08b6ff0e)

  1. use monospace font for feature values
  2. use `name: value = description` syntax
  3. render yaml-style ` - foo` ticks for tree nodes

also, mute ATT&CK ID:
![image](https://github.com/user-attachments/assets/6f79e3e2-e500-4a5b-8d5f-e6d409bd6235)

and mute MBC ID:
![image](https://github.com/user-attachments/assets/ed056836-d9db-4bac-81e7-733b18c9cbfc)

**Comment by mr-tz** [comment]
Agreed! @s-ff have you picked any fonts here specifically yet or these all "default"?

**Comment by s-ff** [comment]
> Agreed! @s-ff have you picked any fonts here specifically yet or these all "default"?

I am using a sigle default font throughtout the entire app: `font-family: Arial, Helvetica, sans-serif;`.

**Comment by s-ff** [comment]
@williballenthin could you check before I close this issue?
